src/state/apis/fiatRamps/selectors.ts

```diff
@@ -34,3 +35,15 @@ export const selectFiatRampBuyAssetsWithMarketData = createSelector(
     }, [])
   },
 )
+export const selectSupportsFiatRampByAssetId = createSelector(
+  selectFiatBuyAssetIds,
+  selectFiatSellAssetIds,
+  selectAssetIdParamFromFilter,
+  selectFiatRampActionFromFilter,
+  (buyAssetIds, sellAssetIds, assetId, fiatRampAction): boolean => {
+    if (!assetId) return false
+    if (!fiatRampAction) return false
+    const targetArray = fiatRampAction === 'buy' ? buyAssetIds : sellAssetIds
```

Comment:
i know i wrote this with you, but i don't think this is exactly what we want.

in our consumption, we're only checking if the asset supports buy, not buy or sell, but the button says buy/sell.

do we want to update this to ignore the action and check both buy and sell asset ids?